pcdsdevices/sequencer.py

```diff
@@ -189,3 +204,135 @@ def stop(self):
         """Stop the EventSequencer"""
         logger.debug("Stopping the EventSequencer")
         self.play_control.put(0)
+
+
+class SequenceLine(Device):
+    """Sub-class for event sequencer line."""
+
+    ec = FCpt(EpicsSignal, '{self.prefix}:EC_{self._ID}:{self._line}')
+    bd = FCpt(EpicsSignal, '{self.prefix}:BD_{self._ID}:{self._line}')
+    fd = FCpt(EpicsSignal, '{self.prefix}:FD_{self._ID}:{self._line}')
+    bc = FCpt(EpicsSignal, '{self.prefix}:BC_{self._ID}:{self._line}')
+    ds = FCpt(EpicsSignal, '{self.prefix}:EC_{self._ID}:{self._line}.DESC')
+
+    def __init__(self, prefix, hutch_id=None, line=None, **kwargs):
+
+        self._ID = hutch_id
+        self._line = line
+
+        super().__init__(prefix, **kwargs)
+
+    def get(self):
```

Comment:
Believe it or not, the default implementation of `ophyd.Device.get` will do this for you - and (arguably) better. 

It gives you not just a tuple of values you have to refer to by index, but it's a `namedtuple` generated automatically of the form `namedtuple('SequenceLineTuple', 'ec bd fd bc ds')`. This means you can access the values either by index `seq.get()[0]` or `seq.get().ec`, for example.

Reply:
I think what he is suggesting here is that if you delete this method it will do the same thing! It will just be a `collections.namedtuple` from the standard library not a `list`.